scripts/update_and_rebuild_libmesh.sh

```diff
@@ -111,6 +111,13 @@ fi
 
 cd $SCRIPT_DIR/../libmesh
 
+# If there is no package in envirment
+# We use petsc submodule
+if [ -z "$PETSC_DIR" ]; then
+  export PETSC_DIR=$SCRIPT_DIR/../petsc
```

Comment:
As discussed, let's emit a warning here so that users know if they build with the submodule they may see some test failures until we officially support PETSc maint